applications/IgaApplication/custom_conditions/support_laplacian_condition.cpp

```diff
@@ -0,0 +1,157 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Nicolò Antonelli
+//                  
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_conditions/support_laplacian_condition.h"
+
+namespace Kratos
+{
+    void SupportLaplacianCondition::CalculateAll(
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo,
+        const bool CalculateStiffnessMatrixFlag,
+        const bool CalculateResidualVectorFlag

```

Comment:
Which is the reasoning behind having these variables? I'd directly implement the `CalculateRightHandSide`, `CalculateLeftHandSide` and `CalculateLocalSystem`.